### ISSUE TITLE
No match state? 

### ISSUE THREAD

**Issue by connectwithraj** [oss6/kamil#2]
Thanks for providing the solution and open sourcing it. It's really helpful. 

Could you please let me know how can I show "no matches" message when there is no entity match in the auto complete. I want users to see some message when there is no match for auto complete. 

Thanks!

**Comment by oss6** [comment]
Hey!
At the moment this feature it's not implemented. However I'll provide it soon, by tomorrow.
Thank you

**Comment by connectwithraj** [comment]
Thanks much! 
Looking forward.

**Comment by oss6** [comment]
Check out the latest version: https://github.com/oss6/kamil/releases/tag/v0.2.1
Now you can add a "no matches found" message as follows:
```javascript
var autoComplete = new Kamil('#tags', {
    source: tags,
    noResultsMessage: 'No results found.',
});
```
When no matches have been found it will show the message as a dropdown.

**Comment by connectwithraj** [comment]
Thanks for quick turnaround!
I tried and it did not work. Also there was no script error either.
I might have to spend more time to see why it did not work.

**Comment by oss6** [comment]
![carbon](https://user-images.githubusercontent.com/4907450/38460998-afa54ae2-3abd-11e8-8100-67e30edaeaad.png)

Have you used it like this? Remember to download the latest release or run `npm install kamil --save`. Bower is not supported anymore.

**Comment by connectwithraj** [comment]
Yes, i downloaded the latest file and I am using it like this 
 var ac = new Kamil('#tags', {
    source: sEntities,
    noResultsMessage: 'No results found.',
    autoFocus: true,
    minChars: 3

sEntities array has around 13k items.

**Comment by oss6** [comment]
In that case the autocomplete will only show the 'No results found' message after 3 characters AND if there is no match. Do you want it to show the message even before the minChars value?

**Comment by oss6** [comment]
By the way, I reproduced your same environment, with 13k items and the options you defined.

**Comment by connectwithraj** [comment]
No. I want it to show "no match" only after at least 3 characters. Is it working for you for 13k items as well ? 

By the way do you think we can connect on Skype or something, If you are interested in seeing the issue . Happy to show you.
